### PR TITLE
Rc 5.2.0 rework examples to uui components

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: uui/components/widgets/ProgressBar.tsx [hunk 1]
```diff
@@ -13,7 +13,10 @@ export function applyProgressBarMods(mods: ProgressBarMods) {
     const size = mods.size || defaultSize;
 
     return [
-        css.root, css[`size-${size}`], mods.striped && css.striped,
+        'uui-progress_bar',
+        css.root,
+        css[`size-${size}`],
+        mods.striped && css.striped,
```
Comment (siarhei-epam) [inline uui/components/widgets/ProgressBar.tsx:19]:
```suggestion
        css.root,
        css[`size-${size}`],
        mods.striped && css.striped,
```
'uui-progress_bar' added to uui-components, look at snapshot

File: app/src/docs/_examples/dnd/Basic.example.tsx [hunk 2]
```diff
@@ -81,10 +79,10 @@ export default function DndMaterial() {
                                 <IconContainer size={ 48 } icon={ FileIcon } />
                             </FlexCell>
                             <FlexCell width="100%" cx={ css.textWrapper }>
-                                <Text cx={ css.text } size="24" lineHeight="24" fontSize="16" font="sans-semibold">
+                                <Text cx={ css.text } size="24" lineHeight="24" fontSize="16" font="semibold">
                                     {item.name}
                                 </Text>
-                                <Text cx={ css.text } size="24" lineHeight="24" fontSize="14" color="gray60">
+                                <Text cx={ css.text } size="24" lineHeight="24" fontSize="14" color="primary">
```
[the remark this answers is not in the document]
Comment (vik753) [reply app/src/docs/_examples/dnd/Basic.example.tsx:85]: fixed

File: app/src/docs/_examples/pickerInput/ConfigurePortalTargetAndPlacement.example.tsx [hunk 3]
```diff
@@ -33,7 +27,7 @@ export default function ConfigurePortalTargetAndPlacement() {
             <Panel>
                 <FlexRow alignItems="top" padding="18" vPadding="24">
                     <FlexCell grow={ 1 } ref={ portalTargetRef }>
-                        <Text fontSize="18" lineHeight="24" color="gray90" font="museo-slab">
+                        <Text fontSize="18" lineHeight="24" color="secondary">
```
[the remark this answers is not in the document]
Comment (vik753) [reply app/src/docs/_examples/pickerInput/ConfigurePortalTargetAndPlacement.example.tsx:30]: fixed

File: app/src/docs/_examples/pickerInput/PickerInputInDropdown.module.scss [hunk 4]
```diff
@@ -1,6 +1,4 @@
-@use '~@epam/promo/assets/styles/colors' as *;
-
 .divider {
-    border-bottom: 1px solid $gray30;
+    border-bottom: 1px solid var(--uui-divider);
```
[the remark this answers is not in the document]
Comment (vik753) [reply app/src/docs/_examples/pickerInput/PickerInputInDropdown.module.scss:2]: fixed

File: app/src/docs/_examples/pickerInput/PickerInputWithCustomFooter.example.tsx [hunk 5]
```diff
@@ -44,7 +42,7 @@ export default function PickerInputWithCustomFooter() {
                         return (
                             <FlexRow padding="12">
                                 <FlexCell width="auto">
-                                    <Text color="gray80">Language Levels according to the CEFR scale</Text>
+                                    <Text color="secondary">Language Levels according to the CEFR scale</Text>
```
[the remark this answers is not in the document]
Comment (vik753) [reply app/src/docs/_examples/pickerInput/PickerInputWithCustomFooter.example.tsx:45]: fixed

File: app/src/docs/_examples/virtualList/Advanced.example.tsx [hunk 6]
```diff
@@ -43,14 +39,14 @@ export default function AdvancedVirtualList() {
                         <li className={ css.mainContainerListItem } key={ row.key + String(row.index) }>
                             {row.value ? (
                                 <>
-                                    <Text font="museo-sans" size="36">
+                                    <Text font="regular" size="36">
```
[the remark this answers is not in the document]
Comment (vik753) [reply app/src/docs/_examples/virtualList/Advanced.example.tsx:42]: fixed

File: uui/components/widgets/ProgressBar.tsx (hunk 1 above)
Comment (vik753) [reply uui/components/widgets/ProgressBar.tsx:19]: fixed

**Review by siarhei-epam — APPROVED** [review]

**Pull request merged**